ankimorphs/filters/am_highlight_morphs.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import annotations
+
+import re
+
+from anki.template import TemplateRenderContext
+
+from ..ankimorphs_config import (
+    AnkiMorphsConfig,
+    AnkiMorphsConfigFilter,
+    get_matching_filter,
+)
+from ..ankimorphs_db import AnkiMorphsDB
+from ..ankimorphs_globals import EXTRA_FIELD_HIGHLIGHTED
+from ..morpheme import Morpheme
+from ..morphemizers import spacy_wrapper
+from ..morphemizers.morphemizer import (
+    Morphemizer,
+    SpacyMorphemizer,
+    get_morphemizer_by_description,
+)
+from ..text_highlighting import get_highlighted_text
+from ..text_preprocessing import (
+    get_processed_morphemizer_morphs,
+    get_processed_spacy_morphs,
+)
+
+
+def am_highlight_morphs(
+    field_text: str,
+    field_name: str,
+    filter_name: str,
+    context: TemplateRenderContext,
+) -> str:
+    """Use morph learning progress to decorate the morphemes in the supplied text.
+    Adds css classes to the output that can be styled in the card."""
+
+    if filter_name != "am-highlight-morphs" or field_name == EXTRA_FIELD_HIGHLIGHTED:
```

Comment:
is "am-highlight-morphs" a little verbose? Maybe "am-highlight" would suffice?

Reply:
agreed; updated.